### PR TITLE
Create a Docker image for AndBible development

### PR DESCRIPTION
**Pull request by uniqueid001**

### CODE REVIEW

**Comment by timbze** [comment]
Is this basically the same idea as #2092?

**Comment by uniqueid001** [comment]
> Is this basically the same idea as #2092?

Yeah, pretty much.  (I wasn't aware of that)  That one builds from the command-line only, while this installs and is able to run Android Studio.

File: docker/README [hunk 1]
```diff
@@ -0,0 +1,37 @@
+About:
+
+AndBible's build system requires a newer version of nodejs than is packaged
+for Ubuntu 20.04.  As a result I can no longer build AndBible.  I created
+this Docker image based on Ubuntu 22.04 so that I can use it to build
+AndBible.
```
Comment (tuomas2) [inline docker/README:6]: Just a tip: you can install any node versions on user level with nvm https://github.com/nvm-sh/nvm

**Review by tuomas2 — APPROVED** [review]

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          curl \
+          expect \
+          figlet \
+          file \
+          git \
+          less \
+          locales \
+          npm \
+          openjdk-18-jdk \
+          openssh-server \
```
Comment (tuomas2) [inline docker/Dockerfile:21]: why installing openssh-server?

Comment (tuomas2) [reply docker/Dockerfile:21]: ah, for X11 forwarding via ssh, okay...

**Pull request merged**